main/res/xml/preferences_map.xml

```diff
@@ -70,6 +70,15 @@
             android:summary="@string/init_summary_renderthemefolder_synctolocal"
             android:title="@string/init_renderthemefolder_synctolocal"
             app:iconSpaceReserved="false" />
+    </PreferenceCategory>
+    <PreferenceCategory
+        android:title="@string/settings_title_map_hillshading"
+        app:iconSpaceReserved="false">
+        <Preference
+            android:key="@string/pref_fakekey_info_offline_maphillshading"
+            android:text="@string/settings_info_hillshading"
```

Comment:
use `android:summary` instead for the text to be displayed

Reply:
I copied it from offline maps and themes info. Guess those are wrong as well then. Should i fix it in this PR?

Reply:
I'm not sure. I've stumbled upon those as well after I'd commented on this PR, so I wonder whether there actually _should_ be an `android:text` attribute within preferences and whether we have only suppressed it somehow. Need to research this a bit first.

Reply:
I opened https://github.com/cgeo/cgeo/issues/13547 for that when I noticed. Didn't have time to look into the cause yet

Reply:
According to https://developer.android.com/develop/ui/views/components/settings/components-and-attributes there's no `text` attribute allowed / recognized within preferences. Maybe the old preferences lib tolerated it, but the AndroidX version does not seem to support it. (Also, during my very short research, I haven't found a single example using that attribute.)
So I guess the existing instances could be renamed to `android:summary`.

Reply:
added a commit replacing all android:text with android:summary to this PR (3 dialogs)